src/librbd/AioImageRequest.h

```diff
@@ -28,13 +29,14 @@ class AioImageRequest {
 
   static void aio_read(ImageCtxT *ictx, AioCompletion *c,
                        Extents &&image_extents, char *buf, bufferlist *pbl,
-                       int op_flags);
+                       int op_flags, const blkin_trace_info *trace_info = nullptr);
```

Comment:
Minor: defaulted params start to get ugly very fast. I'd rather see any compilation errors fixed. This comment applies throughout.